test/AutoDiff/differentiable_attr_type_checking.swift

```diff
@@ -769,3 +769,50 @@ func slope2(_ x: Float) -> Float {
 func slope3(_ x: Float) -> Float {
   return 3 * x
 }
+
+// Index based 'wrt:'
+
+@differentiable(wrt: y) // ok
+func two1(x: Float, y: Float) -> Float {
+  return x + y
+}
+
+ @differentiable(wrt: (x, y)) // ok
```

Comment:
```suggestion
@differentiable(wrt: (x, y)) // ok
```

Same for other occurrences.